tests/test_output_formatter.cpp

```diff
@@ -735,6 +771,39 @@ TEST_F(JsonFormatter, no_images_find_output)
     EXPECT_EQ(output, expected_output);
 }
 
+TEST_F(JsonFormatter, duplicate_images_in_find_output)
+{
+    mp::JsonFormatter formatter;
+    const auto reply = construct_find_multiple_replies_duplicate_image();
+
+    auto expected_output = "{\n"
+                           "    \"errors\": [\n"
+                           "    ],\n"
+                           "    \"images\": {\n"
+                           "        \"core18\": {\n"
+                           "            \"aliases\": [\n"
+                           "            ],\n"
+                           "            \"os\": \"Ubuntu\",\n"
+                           "            \"release\": \"Core 18\",\n"
+                           "            \"remote\": \"\",\n"
+                           "            \"version\": \"20190520\"\n"
+                           "        },\n"
+                           "        \"snapcraft:core18\": {\n"
+                           "            \"aliases\": [\n"
+                           "            ],\n"
+                           "            \"os\": \"\",\n"
+                           "            \"release\": \"Snapcraft builder for core18\",\n"
+                           "            \"remote\": \"snapcraft\",\n"
+                           "            \"version\": \"20190520\"\n"
+                           "        }\n"
+                           "    }\n"
+                           "}\n";
```

Comment:
Just a suggestion, raw string literals might make this nicer on the eye: https://stackoverflow.com/a/5460235